transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -343,6 +359,33 @@ final ChannelFuture initAndRegister() {
 
     abstract void init(Channel channel) throws Exception;
 
+    Collection<ChannelInitializerExtension> getInitializerExtensions() {
+        if (disableExtensions) {
+            return ChannelInitializerExtensions.empty().extensions();
+        }
+        List<ChannelInitializerExtension> extensions = ChannelInitializerExtensions.getExtensions().extensions();
```

Comment:
nit: this code block may be converted to static method: e.g. ChannelInitializerExtensions.filteredExtensions(ApplicableInfo) so code related to extensions is kept is single place. Similar may be done for related parts in Bootstrap, ServerBootstrap

Reply:
I pushed the filtering into the `extensions()` method.